### DISCUSSION TITLE
Question about the screenshot directory

### DISCUSSION THREAD

**Discussion by BryanHoffmaster** [zxch3n/PomodoroLogger#257] [General]
After installing and running this application for a few days, I wanted to dig more into this awesome software. But I noticed that there is a directory that where screenshots are taken at intervals that match the short/long breaks. My question is  what exactly is this used for? I can't find functionality for this inside the application at all. Is there a feature that is coming down the pipe that uses these screenshots? 

If you wouldn't mind, I would like to make a pull request to add an option to clear this directory when the application closes, or maybe store them encrypted in the local db? I don't think keeping around screenshots of your desktop, with potentially secret/sensitive information around in this applications install directory is a good idea.

I'm open to other suggestions on how to handle this, and I would even be open to trying and fixing them myself and sending a pull request your way. 

Let me know what you think!

**Comment by zxch3n** [comment]
I'm open to PR. The screenshot function can be removed totally actually. 

I had some ideas about how we could use the screenshots when I initiated this app. Like we could use OCR to get more context to infer the task we were working on. But an OCR model is heavy and making predictions based on text from the screen is error-prone. And I think not many people need to retrospect on the screenshots as I thought. So it can be safely removed now.
